apiserver/pkg/util/job.go

```diff
@@ -39,6 +47,51 @@ func NewRayJob(apiJob *api.RayJob, computeTemplateMap map[string]*api.ComputeTem
 		}
 		rayJob.Spec.RayClusterSpec = clusterSpec
 	}
+	if apiJob.JobSubmitter != nil {
+		// Job submitter is specified, create SubmitterPodTemplate
+		cpus := "1"
+		memorys := "1Gi"
+		if apiJob.JobSubmitter.Cpu != "" {
+			cpus = apiJob.JobSubmitter.Cpu
+		}
+		if apiJob.JobSubmitter.Memory != "" {
+			memorys = apiJob.JobSubmitter.Memory
+		}
+		rayJob.Spec.SubmitterPodTemplate = &v1.PodTemplateSpec{
+			Spec: v1.PodSpec{
+				Containers: []v1.Container{
+					{
+						Name:  apiJob.Name + "-submitter",
```

Comment:
What's the need to make the container name dynamic? Generally it's preferable to keep them static.

Reply:
To avoid name collision in case we want to run multiple jobs against the same cluster

Reply:
How does the container name related to the ray job submission id?